### ISSUE TITLE
Point 3: Create data with GNN training performance.

### ISSUE THREAD

**Issue by FilipsFedotovs** [FilipsFedotovs/EDER-TSU#43]
Create data with GNN training performance. (Validation Loss, Test loss, Accuracy etc. vs Epochs)
